app/docs/enterprise/0.31-x/admin-gui.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Kong Admin GUI
+class: page-install-method
+---
+
+# Kong Admin GUI
+
+## Access the GUI with SSL enabled
+1. **Configure**
+  
+    The Admin GUI configuration settings in `kong.conf` to enable SSL.
+
+2. **Navigate**
+  
+    Go to your configured Admin GUI `ip:port` using the https:// protocol.
+
+3. **Confirm** 
+  
+    If you are using Kong .30 or newer, make sure you check the `admin_listen` parameter in `kong.conf`. This behavior was changed and by default only listens on the local interface 
```

Comment:
Throughout this entire website and our other public communication channels (Changelog, Kong Nation...), we refer to configuration values such as `admin_listen` as such: "configuration values".

The rationale is that such configuration values can be specified in `kong.conf`, indeed, but also:
1. `kong.conf` can be named something else
2. The configuration file can be in any repository
3. Some distributions do not expose a configuration file by default (i.e. Docker)
4. Each configuration value can be specified as an environment variable (e.g. to configure aforementioned Docker image)

As such, we use the following language to keep interpretation open (and let the user interpret it depending on how they configured their own setup):
```
If you are using Kong 0.30 or newer, make sure that the `admin_listen` configuration value binds to
the desired network interface. By default, and for security reasons, this setting binds to the local interface
only.
```

(I think the rewording makes things clearer too btw)

Reply:
I understand that you may not have originally written those docs, btw - but now that they become public and live alongside with CE docs, we should make an extra effort for consistency imho.